ansible/testbed-cli.sh

```diff
@@ -208,7 +208,8 @@ function start_topo_vms
 
   echo "Starting VMs for testbed '${testbed_name}' on server '${server}'"
 
-  ANSIBLE_SCP_IF_SSH=y ansible-playbook -i $vmfile testbed_start_VMs.yml --vault-password-file="${passwd}" -l "${server}" -e VM_base="$vm_base" -e topo="$topo" $@
+  ANSIBLE_SCP_IF_SSH=y ansible-playbook -i $vmfile testbed_start_VMs.yml --vault-password-file="${passwd}" -l "${server}" \
+	  -e VM_base="$vm_base" -e vm_type="$vm_type" -e topo="$topo" $@
```

Comment:
Skip "add-topo-vms" if "vm_type=='ceos'"?

Reply:
The logic had existed in the beginning of add-topo-vms